### DISCUSSION TITLE
xdr support

### DISCUSSION THREAD

**Discussion by heinono1** [idaholab/moose#21302] [Q&A Getting Started] [answered]
I am (attempting) to install moose without the benefit of being able to pull moose-tools (cannot access inl.gov domain to pull). I get an error trying to install libmesh using the ubpdate_and_install_libmesh.sh script: "XDR was not found but --enable-xdr-required was specified". I have not been able to find out what "xdr" refers to. Where can I find this package or source code and install it?

**Comment by GiudGiud** [comment, marked as answer]
Hello

This is the checkpoint system. Please check these posts for solutions:
https://github.com/idaholab/moose/discussions/20711
https://github.com/idaholab/moose/discussions/16529

Guillaume

**Comment by heinono1** [comment]
Thanks. Yeah I had traced this to libtirpc. We are fring up a new cluster
so all the bits and pieces are not there yet.


Cheers,
Olle

On Tue, Jun 14, 2022 at 4:22 PM Guillaume Giudicelli <
***@***.***> wrote:

> Hello
>
> This is the checkpoint system. Please check these posts for solutions:
> #20711 <https://github.com/idaholab/moose/discussions/20711>
> #16529 <https://github.com/idaholab/moose/discussions/16529>
>
> Guillaume
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/idaholab/moose/discussions/21302#discussioncomment-2950847>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEKZEF3RBE4G6GFIIL7QJALVPDZYZANCNFSM5YY7NZ5A>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Olle Heinonen
***@***.***
